packages/driver/cypress/support/utils.js

```diff
@@ -137,6 +137,36 @@ export const expectCaret = (start, end) => {
   }
 }
 
+export const makeRequestForCookieBehaviorTests = (
```

Comment:
We lost the types associated with this. Can we update utils.js to utils.ts and/or add this as its own typed' file?

Reply:
I was kind of bummed about this. Changing the extension on utils.ts might be a bit of a haul, so I wonder if we just want to create a typed file in `support` with this function in it to preserve the types?

Reply:
I just renamed it it locally, and doesn't seem to be having massive issues.

Reply:
If it isn't very large, I can give it a shot. Main reason I was avoiding it was blowing the scope of this PR up, but if it is fairly simple I don't see why not!